client-app/ui-kit/components/atoms/hexagon-icon/vc-hexagon-icon.vue

```diff
@@ -8,13 +8,14 @@
 import { computed } from "vue";
 
 interface IProps {
-  size?: number;
+  size?: number | string;
```

Comment:
As in https://github.com/VirtoCommerce/vc-theme-b2b-vue/pull/912 I propose to leave only string here for clarity

Reply:
You can make another task for that.

Reply:
https://virtocommerce.atlassian.net/browse/ST-5653